internal/scanner/scanner.go

```diff
@@ -16,9 +16,8 @@ import (
 
 	"github.com/kubewarden/audit-scanner/internal/constants"
 	"github.com/kubewarden/audit-scanner/internal/k8s"
-	reportLogger "github.com/kubewarden/audit-scanner/internal/log"
 	"github.com/kubewarden/audit-scanner/internal/policies"
-	"github.com/kubewarden/audit-scanner/internal/report"
+	report "github.com/kubewarden/audit-scanner/internal/report"
```

Comment:
silly question, do we really need to rename this import? It should be available as `report` without having to do that

Reply:
I blame goimports for this 😇